roles/letsencrypt/meta/main.yml

```diff
@@ -0,0 +1,21 @@
+dependencies:
+  - role: apache
+    ssl: no
+
+    apache_mods_enabled:
+      - rewrite.load
+
+    apache_vhosts:
```

Comment:
So, we're probably going to hit a 1 off downtime, and we'll need to update our webhook urls anyway. 

From an ongoing perspective, why do we need to merge them? Won't they just continue to drop vhosts on seperate ports/